timesketch/lib/datastores/elastic.py

```diff
@@ -221,6 +224,19 @@ def search(
         DEFAULT_LIMIT = 500  # Maximum events to return
         LIMIT_RESULTS = query_filter.get(u'limit', DEFAULT_LIMIT)
 
+        # Default timeout for the scroll API
+        default_scroll_timeout = u'1m'  # 1 minute
+        if scroll:
```

Comment:
I would make the distinction between the boolean "enable_scroll" parameter and the string "scroll_timeout" to pass down to the client below.
Otherwise LGTM =)